boards/arm/stm32l562e_dk/Kconfig.defconfig

```diff
@@ -45,4 +46,74 @@ config USE_DT_CODE_PARTITION
 
 endif # TRUSTED_EXECUTION_NONSECURE
 
+
+if LVGL
+
+config DISPLAY
+	default y
+
+config LVGL_DISPLAY_DEV_NAME
+	default "DISPLAY"
+
+config LVGL_HOR_RES_MAX
+	default 240
+
+config LVGL_VER_RES_MAX
+	default 240
+
+config LVGL_VDB_SIZE
+	default 16
+
+config LVGL_BITS_PER_PIXEL
+	default 16
+
+choice LVGL_COLOR_DEPTH
+	default LVGL_COLOR_DEPTH_16
+endchoice
+
+config KSCAN
+	default y
+
+config LVGL_POINTER_KSCAN
+	default y
+
+config LVGL_POINTER_KSCAN_DEV_NAME
+	default "FT5336"
+
+# Original orientation of the display needs to invert Y only
+
+config LVGL_POINTER_KSCAN_SWAP_XY
+	default n
+
+config LVGL_POINTER_KSCAN_INVERT_X
+	default n
```

Comment:
Why this ?

This is already a `default n` here:
https://github.com/zephyrproject-rtos/zephyr/blob/9c74027a7bbdc7348441ef2144a8c198c4e13d45/lib/gui/lvgl/Kconfig.input#L33-L35

Reply:
That's the same reason as your previous comment :)

Reply:
Based on your suggestions, I have removed these config lines and rely on the default values of the LVGL library.